toolhub_auth/templates/auth/login.jinja

```diff
@@ -0,0 +1,20 @@
+{% extends "layouts/base.jinja" %}
+
+{% block head_title %}{{ _("Login") }}{% endblock %}
+
+{# no page header yet #}
+{% block page_header %}
+  <h2 class="mb-0">{{ _("Login") }}</h2>
+{% endblock %}
+
+{% block content %}
+<div class="row">
```

Comment:
Have some sort of header or call-to-action. It's not immediately clear that I was redirected to the login page whenI clicked on the Tools link. Would probably be a good idea throughout the site.

Reply:
Will do.